### ISSUE TITLE
lightningd: --jrest-on-startup: unrecognized option

### ISSUE THREAD

**Issue by jsarenik** [clightning4j/clightning4j-node#7]
```
$ git rev-parse HEAD
b6cca9deafd4c51029f5ffb05c22cca53dc30b81
$ cd 0.10.1
$ docker-compose -f rest-mode-docker-compose.yml up
...
Attaching to 0101_lightningd_1
lightningd_1  | /entrypoint.sh: assuming arguments for lightningd
lightningd_1  | /entrypoint.sh: setting data directory to /home/clightning4j/.lightning
lightningd_1  | 
lightningd_1  | 2021-08-25T14:56:53.869Z UNUSUAL lightningd: Creating configuration directory /home/clightning4j/.lightning/bitcoin
lightningd_1  | 2021-08-25T14:56:53.869Z INFO    plugin-manager: /opt/clightning-v0.10.0/bin/../libexec/c-lightning/plugins/bcli: disabled via disable-plugin
lightningd_1  | 2021-08-25T14:56:53.869Z UNUSUAL lightningd: Creating configuration directory /home/clightning4j/.lightning/bitcoin
lightningd_1  | 2021-08-25T14:56:53.869Z DEBUG   plugin-manager: blacklist for bcli
lightningd_1  | 2021-08-25T14:56:53.869Z INFO    plugin-manager: /opt/clightning-v0.10.0/bin/../libexec/c-lightning/plugins/bcli: disabled via disable-plugin
lightningd_1  | lightningd: --jrest-on-startup: unrecognized option
0101_lightningd_1 exited with code 1
```

**Comment by vincenzopalazzo** [comment]
Hi @jsarenik, maybe we can include this fix inside the following PR https://github.com/clightning4j/clightning4j-node/pull/6

**Comment by vincenzopalazzo** [comment]
Fixed in version 0.10.1 of the docker image
